sound/soc/sof/intel/apl.c

```diff
@@ -91,6 +91,10 @@ struct snd_sof_dsp_ops sof_apl_ops = {
 	.pre_fw_run = hda_dsp_pre_fw_run,
 	.post_fw_run = hda_dsp_post_fw_run,
 
+	/* dsp core power up/down */
+	.core_power_up = hda_dsp_enable_core,
+	.core_power_down = hda_dsp_core_reset_power_down,
```

Comment:
You want to add the implementation in a later patch, otherwise you break git bisect. This patch doesn't compile without additional ones, that's not good.